packages/helix-shared-indexer/test/index-resource.test.js

```diff
@@ -149,6 +157,8 @@ const BODY = `
   </div>
   <div class="default">
     <p>Topics: A, B, C,</p>
+    <p>before<br><br>after</p>
+    <p>before<br>after</p>
```

Comment:
the first <p> will match the first alternate in `first-alternate`
the second <p> will match the second alternate in `second-alternate`

which would be less confusing if we had more than 1 document to index ;-)